resources/migrations/001_update_migrations.yaml

```diff
@@ -10079,6 +10079,64 @@ databaseChangeLog:
         - customChange:
             class: "metabase.db.custom_migrations.CreateSampleContentV2"
 
+  - changeSet:
+      id: v52.2024-12-10T10:00:00
+      author: qnkhuat
+      comment: add notification.internal_id
+      preConditions:
+        - not:
+          - columnExists:
+              tableName: notification
+              columnName: internal_id
+      changes:
+        - addColumn:
+            tableName: notification
+            columns:
+              - column:
+                  name: internal_id
+                  type: varchar(254)
+                  remarks: the internal id of the notification
+                  constraints:
+                    nullable: true
+                    unique: true
+
+  - changeSet:
+      id: v52.2024-12-10T10:00:10
+      author: qnkhuat
+      comment: Truncate the notification tables
```

Comment:
I think this deserves a motivating comment about how this is a preparatory step for not needing to truncate again.

Reply:
added one